gulpfile.js

```diff
@@ -170,12 +170,12 @@ function makeWebpackPkg() {
     .pipe(webpackStream(cloned, webpack))
     .pipe(uglify())
     .pipe(replace(/('|")v\$prebid\.modulesList\$('|")/g, makeModuleList(externalModules)))
-    .pipe(gulpif(file => file.basename === 'prebid-core.js', header(banner, { prebid: prebid})))
+    .pipe(gulpif(file => file.basename === 'prebid-core.js', header(banner, { prebid: prebid })))
```

Comment:
Technically, this would also be a valid syntax:

```suggestion
    .pipe(gulpif(file => file.basename === 'prebid-core.js', header(banner, { prebid })))
```